### DISCUSSION TITLE
Video not rendering in InShot app

### DISCUSSION THREAD

**Discussion by hentsch** [LSPosed/MagiskOnWSALocal#355] [General]
Hi, 
I don't know if it is the right place to ask my question. If not, ignore it and please, let me know where I can search for the answer.

I'm using WSA, installed from this repo. 
I try to use the InShot app to edit some video. The app load well. But the video doesn't render, there is a black window instead of the video.
![image](https://user-images.githubusercontent.com/15311685/212725690-b55dc014-aefa-46c7-a8cd-cbdace586325.png)

The same appends with LightCut.

Any idea ?
Thanks
Vince

**Comment by Howard20181** [comment]
Report to Microsoft

**Reply by hentsch** [reply]
Thanks @Howard20181 for your answer !
Neither InShot nor LightCut are present in the Amazon Appstore, so Microsoft probably won't reply, as it's not officially supported.
That's why I asked my question here...
Anyway, I'll report to MS
Regards
